test_scripts/Polices/build_options/016_ATF_P_TC_Timeout_Countdown_Start_PTU_Response_HTTP.lua

```diff
@@ -122,9 +119,9 @@ function Test:TestStep_Sending_PTS_to_mobile_application()
 
   EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "HTTP", fileType = "JSON"})
```

Comment:
@istoimenova, According to comment in clarification "OnSystemRequest and service type" fileType has to be "BINARY". Please correct.